frontend/server/k8s-helper.ts

```diff
@@ -133,7 +190,8 @@ export async function getTensorboardInstance(
         if (
           viewer &&
           viewer.body &&
-          viewer.body.spec.tensorboardSpec.logDir === logdir &&
+          viewer.body.spec.tensorboardSpec.logDir ===
```

Comment:
It's not very obvious why podTemplateSpec is a needed argument for upstream callers. I think it's okay we just skip checking logDir here matches. The most important check is already done by `getNameOfViewerResource`.

So it's okay to remove the extra argument added to many other methods.

Reply:
`getNameOfViewerResource` generate hashcode for `logdir`,  this check just in case of Hash Collision

Reply:
Thanks, that's a good point!
But I'd still prefer removing the argument and log a TODO here for hash collision.

My concerns:
1. this extra argument is very confusing for method signature and appeared in many places.
2. if there is a hash collision, the system breaks anyway...
3. typically number of tensorboard instances is very very small, so risk is very low.